srcpkgs/qbe/template

```diff
@@ -9,15 +9,21 @@ maintainer="Bryce Vandegrift <bryce@brycevandegrift.xyz>"
 license="MIT"
 homepage="https://c9x.me/compile/"
 distfiles="https://c9x.me/compile/release/qbe-${version}.tar.xz"
-checksum=7d0a53dd40df48072aae317e11ddde15d1a980673160e514e235b9ecaa1db12c
+checksum=a6d50eb952525a234bf76ba151861f73b7a382ac952d985f2b9af1df5368225d
 
 # Currently only aarch64 and x86_64 targets are supported and the checks
 # test the compiled binaries.
 case "$XBPS_TARGET_MACHINE" in
 	aarch64*|x86_64*) ;;
-	*) make_check=no ;;
+	*) make_check=no # tests are broken ;;
```

Comment:
Tests exist for riscv64 and aarch64, but they can only be ran with the host qbe and I wasnt sure if it would work in the github workflow, but I tested locally and it looks like it doesnt run tests for target arch when you are cross compiling.
```
check-arm64: qbe
        TARGET=arm64 tools/test.sh all

check-rv64: qbe
        TARGET=rv64 tools/test.sh all
 ```
 I updated the template and I think its OK now.